fulp_modules/features/ert/code/safety_moth.dm

```diff
@@ -0,0 +1,185 @@
+
+	skillchips = null
+
+/// Clothing
+// ID
+/obj/item/card/id/advanced/centcom/ert/safety_moth
+	registered_name = "Safety Moth"
+	trim = /datum/id_trim/centcom/ert/commander
+// Headset
+/datum/outfit/centcom/ert/safety_moth/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
+	. = ..()
+	if(visualsOnly)
+		return
+	var/obj/item/radio/R = H.ears
+	R.keyslot = new /obj/item/encryptionkey/heads/captain
+	R.recalculateChannels()
+
+//Eng
+/obj/item/clothing/suit/space/safety_moth
+	name = "Safety Moth safety vest"
+	desc = "The standard issue safety vest of the Safety Moth. Functions like a hardsuit and offers superb protection against environmental hazards."
+	icon_state = "hazard"
+	inhand_icon_state = "hazard"
+	flags_inv = null
+	armor = list(MELEE = 50, BULLET = 40, LASER = 40, ENERGY = 50, BOMB = 50, BIO = 100, RAD = 100, FIRE = 100, ACID = 80)
+	allowed = list(/obj/item/gun, /obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/melee/baton, /obj/item/restraints/handcuffs, /obj/item/tank/internals, /obj/item/toy/plush/moth)
+	resistance_flags = FIRE_PROOF
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+	slowdown = 0
+/obj/item/clothing/head/helmet/space/safety_moth
+	name = "Safety Moth safety hardhat"
+	desc = "The standard issue safety hardhat belonging to Safety Moth themselves. Functions like a hardsuit helmet and offers superb protection against environmental hazards."
+	icon_state = "hardhat0_yellow"
+	resistance_flags = FIRE_PROOF
+	armor = list(MELEE = 50, BULLET = 40, LASER = 40, ENERGY = 50, BOMB = 50, BIO = 100, RAD = 100, FIRE = 100, ACID = 80)
+	heat_protection = HEAD
+	max_heat_protection_temperature = FIRE_IMMUNITY_MAX_TEMP_PROTECT
+/obj/item/clothing/shoes/magboots/safety_moth
+	desc = "A special pair of Magnetic boots which doesn't weight you down." // Do they even need this? They're a moth ffs.
+	name = "safety magboots"
+	slowdown_active = 1
+//Med
+/obj/item/clothing/suit/space/safety_moth/med
+	name = "Safety Moth medical jacket"
+	desc = "The standard issue safety jacket belonging to Safety Moth themselves. Functions like a hardsuit and offers superb protection against environmental hazards."
+	icon_state = "labcoat_paramedic"
+	inhand_icon_state = "labcoat_paramedic"
+/obj/item/clothing/head/helmet/space/safety_moth/med
+	name = "Safety Moth medical cap"
+	desc = "The standard issue safety cap belonging to Safety Moth themselves. Functions like a hardsuit helmet and offers superb protection against environmental hazards."
+	icon_state = "paramedicsoft"
+//Sec
+/obj/item/clothing/suit/space/safety_moth/sec
+	name = "Safety Moth riot suit"
+	desc = "The standard issue riot suit belonging to Safety Moth themselves. Functions like a hardsuit, resists shoves, and offers superb protection against environmental hazards."
+	icon_state = "riot"
+	inhand_icon_state = "riot"
+	clothing_flags = BLOCKS_SHOVE_KNOCKDOWN
```

Comment:
this overwrites the space suit's clothing flags, resulting in a suit without pressure protection or without being thick.
Yes I made the same error with the sec commander ert hardsuit